Makefile

```diff
@@ -71,9 +71,22 @@ setup: ### Setup remote environment
 	for file in $(PROJECT_FILES); do $(NEURO) cp ./$$file $(PROJECT_PATH_STORAGE)/$$file; done
 	$(NEURO) exec --no-tty --no-key-check $(SETUP_JOB) "bash -c 'export DEBIAN_FRONTEND=noninteractive && $(APT) update && cat $(PROJECT_PATH_ENV)/apt.txt | xargs -I % $(APT) install --no-install-recommends % && $(APT) clean && $(APT) autoremove && rm -rf /var/lib/apt/lists/*'"
 	$(NEURO) exec --no-tty --no-key-check $(SETUP_JOB) "bash -c '$(PIP) -r $(PROJECT_PATH_ENV)/requirements.txt'"
+ifdef __BAKE_SETUP
+	make __bake
+endif
 	$(NEURO) --network-timeout 300 job save $(SETUP_JOB) $(CUSTOM_ENV_NAME)
 	$(NEURO) kill $(SETUP_JOB)
 
+.PHONY: __bake
+__bake: upload-code upload-data upload-notebooks
+	echo "#!/usr/bin/env bash" > jupyter.sh
+	echo "jupyter notebook --no-browser --ip=0.0.0.0 --allow-root \
```

Comment:
considering this pr, let's get rid of this code?
https://github.com/neuromation/neuro-base-environment/pull/91

Reply:
no, that one is not merged nor released.

Reply:
it's green, so we can do both right today

Reply:
I'd rather do this in a separate PR.